### PR TITLE
Add Github Actions Workflow to Run `unittest` and Generate Coverage Badge

### PR DESCRIPTION
**Pull request by JoeStanleySEL**
Not sure how terribly important this is, but I find these sorts of automated workflows to be helpful in terms of general project visibility. This documents the basics of running tests manually, as well as automating the process with a Github action that also generates a code-coverage badge.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Bitbucket Server API Wrapper
 A simple wrapper for the Atlassian's Bitbucket Server / Bitbucket Datacenter (formerly Stash) REST API, written in Python.
 
+[![Test](https://github.com/JoeStanleySEL/bitbucketserver/actions/workflows/test-and-coverage.yml/badge.svg?branch=master)](https://github.com/JoeStanleySEL/bitbucketserver/actions/workflows/test-and-coverage.yml)
```
Comment (JasonKempSEL) [inline readme.md]: These links should be to this repo; can we just change the paths to our org and they will work once the scripts run?

Comment (JoeStanleySEL) [reply readme.md]: Absolutely!

I'd only used reference to my personal repo to provide some form of demonstration of how they might look:
image

Comment (JoeStanleySEL) [reply readme.md]: Resolving conversation because this change has been made.

**Comment by JoeStanleySEL** [comment]
@JasonKempSEL: Ready for another look, thank you!

**Pull request merged**